.gitignore

```diff
@@ -1,24 +1,23 @@
-#This file is generated by ModuleSync, do not edit.
-pkg/
-Gemfile.lock
-Gemfile.local
-vendor/
-spec/fixtures/manifests/
-spec/fixtures/modules/
-log/
-junit/
-.vagrant/
+#This file is generated by ModuleSync, do not edit.Z
```

Comment:
Additional `Z` - intentional?

Reply:
https://github.com/puppetlabs/modulesync_configs/pull/170/files  way ahead of you :) thanks to @willmeek for spotting it too.